velox/exec/tests/utils/PlanBuilder.h

```diff
@@ -811,6 +811,12 @@ class PlanBuilder {
     return *this;
   }
 
+  /// Set file format.
```

Comment:
Perhaps, clarify that is used as the default format in table scan and table write. But... I see that TableScan doesn't use it because the format is specified as part of the split, not the plan node. If that's the case, then TableWrite is the only plan node that can use this and there can be only one of these in the plan... hence, maybe add this as a parameter to tableWrite() method?

```
/// Set default file format to use in TableScan and TableWrite plan nodes.
```

Reply:
I agree. Added as an argument to `tableWriter()` API.